docs/source/get_started/issues.rst

```diff
@@ -13,22 +13,22 @@ broken.
 *Note: This list is likely to be non-exhaustive. If we missed something, 
 please let us know [here](https://github.com/deepchem/deepchem/issues/2376).*
 
-+--------------------------------+-------------------+---------------------------------------------------+
-| Feature                        | Deepchem response | Tracker and notes                                 |
-|                                |                   |                                                   |
-+================================+===================+===================================================+
-| AtomicConvModel                | Severe            | [pull#2374](https://github.com/deepchem/deepchem/pull/2374)                   |
```

Comment:
We can remove both `AtomicConvModel` and `RdkitGridFeaturizer` thanks to @ncfrey's hard work!

Reply:
Oh my gosh AtomicConv is fixed? That's amazing! There's a couple of issues and PRs we could close then